### ISSUE TITLE
VST3 realtime effect dialog - plain gui - confusing bugs in the default button behaviour

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#3572]
### Steps to reproduce

Open a VST3 effect in plain gui mode.

Case A:
1. Press Enter.

Case B:
1. Press Shift + tab, then tab.
2. Press Enter.

Case C:
1. Press Shift + tab three times.
2. Press Enter.

### Expected behavior

I don't know the design spec for this, but I presume that there isn't meant to be a default button, and so pressing Enter when a check box or a slider is the focus should do nothing, and there should be no visual indication that a button is the default button.

### Actual behavior

Case A:
![image](https://user-images.githubusercontent.com/11887021/189092112-41e1e22d-8002-4108-ada0-d7dd595f4259.png)
Nothing happens.

Case B:
![image](https://user-images.githubusercontent.com/11887021/189092424-71e8b4a0-108d-424b-baf6-1b71480c3675.png)
Presets & Settings menu opens, despite no visual indication that the corresponding button is the default button.

Case C:
![image](https://user-images.githubusercontent.com/11887021/189092859-6615bae3-8216-4f1c-89c4-3706d8c40467.png)
The Power toggle button is toggled, despite the visual indication that the Presets & Settings button is the default button.

### Audacity Version

current master / alpha / nightly build

### Operating system

Windows

**Comment by LWinterberg** [comment]
Hello! We just [switched development to Audacity 4](https://forum.audacityteam.org/t/master-has-been-switched-to-4-0-0-development/120444). We've redone, or plan on redoing, various things from scratch, including the entire UI, and the part this bug is about. As such, I'm closing this bug as it will be fixed there and we won't be fixing it for the final releases of Audacity 3.
